__tests__/app.test.js

```diff
@@ -82,4 +80,19 @@ describe('GET /api/articles/:article_id', () => {
             expect(body.msg).toBe("Invalid input")
         })
     })
+})
+describe('GET /api/articles', () => {
+    it('returns status 200 and list of all articles in test DB in descending order', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                const articleKeys = ['author', 'title', 'article_id', 'topic', 'created_at', 'votes', 'article_img_url', 'comment_count']
+                expect(body.articles).toBeSortedBy('created_at', { descending: true })
+                body.articles.forEach((article) => {
```

Comment:
We should have a length check here before we forEach through.
We might get a false positive if we get back an empty array